### ISSUE TITLE
About training data normalization

### ISSUE THREAD

**Issue by WUMINGCHAzero** [OpenRobotLab/AnySplat#21]
I notice that the co3d dataset code normalizes the scene by `make_baseline_1` while vggt paper says it normalizes the scene by average distance of points to the origin (the origin of the first camera). The training code has normalize_by_pts_3d option, but is set to false in experiment configs, and I'm curious about why.

Thanks for your answer!

**Comment by jianglh-WHU** [comment]
In the training process, we don't need GT poses; we only need GT RGB images. Therefore, we don't care about the normalization function or whether `normalize_by_pts_3d` is set to true or false.

**Comment by WUMINGCHAzero** [comment]
Thanks! The depth consistency loss is used, does that need normalization?

**Comment by WUMINGCHAzero** [comment]
Thanks!
The depth consistency loss and depth_distillation loss seem to need normalization? Besides, there is also distill _loss for camera pose

**Comment by jianglh-WHU** [comment]
We distill depth and pose from a frozen VGGT. The depth maps and the camera poses from VGGT have already been normalized.
